src/components/profile/loansAndReservationsPage/Page.js

```diff
@@ -5,7 +5,7 @@ import styles from "./LoansAndReservations.module.css";
 import Translate from "@/components/base/translate";
 import ProfileLayout from "../profileLayout";
 import Text from "@/components/base/text";
-import { encodeString } from "@/lib/utils";
+import { encodeString, extractCreatorPrioritiseCorporation } from "@/lib/utils";
```

Comment:
Jeg har vist navngivet `extractCreatorPrioritiseCorporation`.

Skal der ikke så `creators` i stedet for `creator`?

Reply:
dataReducer kalder den og giver creator som prop nedaf. Så underkomponenter ikke skal bekymre sig om hvor data kommer fra

Reply:
Hov, jeg manglede også lidt af min besked.... Jeg mente:

Jeg har vist navngivet `extractCreatorPrioritiseCorporation` forkert.

Skal der ikke stå `extractCreatorsPrioritiseCorporation` (`extractCreators...` i stedet for `extractCreator...`